CMakeLists.txt

```diff
@@ -1,14 +1,17 @@
 #############################################################
 # CMake settings
 cmake_minimum_required(VERSION 3.5)
+cmake_policy(SET CMP0071 NEW)
+cmake_policy(SET CMP0074 NEW)
 
+set(QML_IMPORT_PATH ${CMAKE_SOURCE_DIR}/src/qml/imports CACHE PATH "")
```

Comment:
What's the reason for this? I can't see it used anywhere

Reply:
Qt Creator reports that it can't find `Theme` QML module, thus QML designer doesn't work. If this doesn't happen to you then I guess it's a local problem.

Reply:
I.e. this improves Qt Creator properly detecting the `Theme` module?
That sounds really nice!
Can you add a short comment that this is for Qt Creator IDE support?